### ISSUE TITLE
MetaboAnalyst sets documentation

### ISSUE THREAD

**Issue by tetomonti** [montilab/hypeR-db#2]
Did you record somewhere the procedure you used to download these sets? I mean the actual code?

**Comment by cbragdon93** [comment]
Yes, I have the actual code still on my person. Wrapping up and reviewing the docs for it currently

**Comment by cbragdon93** [comment]
updated my github repo that I've been working with, and sent an invite to the account you're currently using on this page now. In case it didn't go through, [here's the link](https://github.com/cbragdon93/metaboAnalystLibraries)
